### PR TITLE
MM-1899 Added help text to channel notifications

### PR DESCRIPTION
**Pull request by nickago**

### CODE REVIEW

File: web/react/components/setting_item_max.jsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default class SettingItemMax extends React.Component {
 
         var extraInfo = null;
         if (this.props.extraInfo) {
-            extraInfo = this.props.extraInfo;
+            extraInfo = (<div className='setting-list__hint'>{this.props.extraInfo}</div>);
```
Comment (rgarmsen2295) [inline web/react/components/setting_item_max.jsx]: Will this break anything that was already passing in extraInfo as a prop without the assumption of being inclosed in the new div?

Comment (nickago) [inline web/react/components/setting_item_max.jsx]: No, because the only only thing using this prop was the the code I took out

Comment (rgarmsen2295) [inline web/react/components/setting_item_max.jsx]: Gotcha thanks! Awesome, everything looks good then.

**Comment by rgarmsen2295** [comment]
+1, looks good

**Comment by crspeller** [comment]
Interesting that the help text was the exact opposite of helpful.  +1

**Pull request merged**